shortestBridge.js

```diff
@@ -0,0 +1,187 @@
+/*
+We're going to approach this problem with a multi-source BFS. The grid will contain
+two islands, so we can first gather all the locations of the first island only.
+
+Once we have the locations of the first island, we can perform our BFS by adding 
+them to the queue with a starting path of 0. Since we're trying to find the shortest path 
```

Comment:
Since we're using a multi-sourced BFS, we can start from multiple locations because the source's path will always start at 0. And as we BFS/traverse, we only update the path if it's only the first time visiting that location (in this case the second island). With this approach, we'll have the ideal path.

Reply:
This is correct. The answer I was looking for is that we start from every single location of the first island.